parla/task_runtime.py

```diff
@@ -1350,10 +1361,15 @@ def __init__(self, environments: Collection[TaskEnvironment], n_threads: int = N
         self._ready_queue = deque()
 
         # The device queues where scheduled tasks go to be launched from
-        self._device_queues = {dev: deque() for dev in self._available_resources.get_resources()}
+        self._compute_task_dev_queues = {dev: deque() for dev in self._available_resources.get_resources()}
+        self._datamove_task_dev_queues = {dev: deque() for dev in self._available_resources.get_resources()}
```

Comment:
Do we need queues for the CPUs?

Reply:
For now, we don't need it. Do you think is having it overhead? Depending on our policy, it might require in the future. But I have no strong preferences.